pom.client.xml

```diff
@@ -246,6 +246,12 @@
             <scope>test</scope>
          </dependency>
 
+         <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-simple</artifactId>
```

Comment:
This needs changing to `slf4j-api` also. Soon it won't be a test scope dependency either - probably makes sense to just put it in now as not being test scope (it can still be test scoped in the azconfig pom).

Reply:
Oops. my ctrl+H missed that. Fixed. Also made it a normal dependency rather than just a test.